mk/config.mk

```diff
@@ -206,10 +206,20 @@ CFG_WITH_USER_TA ?= y
 # $(ta-targets).
 
 # Load user TAs from the REE filesystem via tee-supplicant
-# There is currently no other alternative, but you may want to disable this in
-# case you implement your own TA store
 CFG_REE_FS_TA ?= y
 
+# Pre-authentication of TA binaries loaded from the REE filesystem
+#
+# - If CFG_REE_FS_TA_BUFFERED=y: load TA binary into a temporary buffer, check
+#   the signature, then process the file only if it is valid.
+# - If disabled: hash the binaries as they are being processed and verify the
+#   signature as a last step.
+ifneq ($(CFG_WITH_PAGER),y)
```

Comment:
Actually the TA is buffered in the so-called TA_RAM, that is not the small TEE_RAM where pager lives.
Therefore I think we can relax this and enable `CFG_REE_FS_TA_BUFFERED` even if `CFG_WITH_PAGER=y`.

Reply:
Well I have failures in `xtest 1013` (large TA)` when `CFG_WITH_PAGER=y CFG_REE_FS_TA_BUFFERED=y` on HiKey960.

Reply:
Maybe I misunderstood your commet but I think xtest 1013 for large TA do fail with `CFG_WITH_PAGER=y`, whatever `CFG_REE_FS_TA_BUFFERED` configuration.
Looking a the code, I see no strong reason for `CFG_REE_FS_TA_BUFFERED=y` to consume much more TEE_RAM paged or unpaged memory than when disabled.

Reply:
On HiKey960, `xtest 1013` works with `CFG_WITH_PAGER=y CFG_REE_FS_TA_BUFFERED=n` and fails with `CFG_WITH_PAGER=y CFG_REE_FS_TA_BUFFERED=y`.

Reply:
Ok, I will have a look when back. Maybe i missed something...

Reply:
Note that I see no issue merging the feature as currently proposed in this P-R (i mean being default enabled when pager is disabled).

Reply:
I tested on Qemu for armv7 with `CFG_WITH_PAGER=y CFG_REE_FS_TA_BUFFERED=y`, `xtest 1013` is ok.

Reply:
Yes, I know. It's Hikey960 that fails.

Reply:
any idea how this could relate to the pager? as an out-of-memory failure?

Reply:
No I have not investigated in details, I can check tomorrow.

Reply:
@etienne-lms my mistake, see comments in fixup patch.